README.md

```diff
@@ -51,7 +51,7 @@ You can take a look at the original project proposal [here](https://github.com/a
 
 Fork (if intending to make changes) or clone this repository.
 
-To start the front end of the project, navigate to the front-end folder and run the following commands in the command line:
+To start the front end of the project, navigate to the front-end folder and run the following command in the command line:
```

Comment:
I think this still would work well as "commands" rather than "command"? Since I view this sentence as applying to everything until `npm start`. Or we could just change it to "do the following" and before `npm install` and `npm start` we make sure to specify those are run in the command line in the `front-end` folder

Reply:
Makes sense, will change